html_css/css/style.css

```diff
@@ -983,12 +1050,21 @@ button:active {
 }
 
 .blog {
-  max-width: 1440px;
+  padding: 82px 0px 0px 0px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly
+}
 
+.blog__slider_item {
```

Comment:
Have a look on BEM again. _ - used for Modifiers, not more nested items

Reply:
Done